### PR TITLE
Update github-script action to v6

### PR DESCRIPTION
**Pull request by luislera**
### Describe the issue
When using the actions/github-script action, a deprecation warning is shown.

imagen

### Changes included in the PR

- Updated the action version to v6.

### Screenshots

After updating the action's version the warning is not shown anymore.

imagen

### CODE REVIEW

**Review by devkeydet — APPROVED** [review]

**Comment by devkeydet** [comment]
Feel free to squash merge your changes into main.  I prefer the author merge changes once the PR is approved so they get credit for the commit.

**Comment by luislera** [comment]
> Feel free to squash merge your changes into main. I prefer the author merge changes once the PR is approved so they get credit for the commit.

Hey Marc, the merge button is not available due to lack of permission.

**Pull request merged**
